config/esw.yaml

```diff
@@ -113,38 +113,38 @@ brushed_motors:
   test: false
   
 cameras:
-    arguments:
-      # this has [bps, width, height, fps]
-      worst_res:
-        - "173000"
-        - "320"
-        - "240"
-        - "15"
-      low_res:
-        - "691000"
-        - "640"
-        - "480"
-        - "15"
-      medium_res:
-        - "2000000"
-        - "960"
-        - "720"
-        - "15"
-      high_res:
-        - "3000000"
-        - "1280"
-        - "720"
-        - "15"
-      best_res:
-        - "4200000"
-        - "1280"
-        - "720"
-        - "30"
-    max_video_device_id_number: 10
-    ips:
-      primary: "10.0.0.7"
-      secondary: "10.0.0.8"
-    skip_every_other_device: false
+  arguments:
+    - bps: "173000"
```

Comment:
u brought up the point earlier, but we probably dont need to make most of these strings